### PR TITLE
addition of web dev resources

### PR DESCRIPTION
**Pull request by Mannuel25**
I added more web dev resources

### CODE REVIEW

**Review by chryzcode — COMMENTED** [review]
@namangupta1399, Eke or Favourite

Please review this changes....the links

Thanks

File: README.md [hunk 1]
```diff
@@ -31,8 +31,22 @@ You can create segments for html blogs, documentations, repositories, books and
 </summary>
 <ul>
     <li><a href="https://html5doctor.com"> HTML5 Doctor</a></li>
+    <li><a href="https://progate.com/courses/html">Progate</a></li>
     <li><a href="https://html5rocks.com"> HTML5 Rocks</a></li>
```
Comment (namangupta1399) [inline README.md:35]: HTML5 Rocks is now **Web Fundamentals by Google Developers**
New link: https://developers.google.com/web

Comment (namangupta1399) [inline README.md:35]: HTML5 Rocks is now **Web Fundamentals by Google Developers**
New link: https://developers.google.com/web

Comment (namangupta1399) [inline README.md:35]: HTML5 Rocks is now **Web Fundamentals by Google Developers**
New link: https://developers.google.com/web

**Comment by namangupta1399** [comment]
@chryzcodez , LGTM 👍

**Review by chryzcode — APPROVED** [review]

**Pull request merged**